test/build/50_dockerfile.bats

```diff
@@ -197,3 +197,105 @@ EOF
         [[ $status -eq 0 ]]
     fi
 }
+
+@test 'Dockerfile: COPY errors' {
+    scope standard
+    [[ $CH_BUILDER = none ]] && skip 'no builder'
+    [[ $CH_BUILDER = buildah* ]] && skip 'Buildah untested'
+
+    # Dockerfile on stdin, so no context directory.
+    if [[ $CH_BUILDER != ch-grow ]]; then  # ch-grow doesn't support this yet
```

Comment:
What does this mean RE ch-grow doesn't support this yet?

Reply:
If you give Docker a Dockerfile on stdin without `-f`, it has no context directory. `ch-grow` doesn't support not having a context directory.

Reply:
Ah, okay.